integration/js/ContentShareVideoTest.js

```diff
@@ -0,0 +1,58 @@
+const {ClickContentShareVideoTestButton, ClickContentShareButton} = require('./steps');
+const {ContentShareVideoCheck} = require('./checks');
+const {RosterCheck} = require('./checks');
+const {TestUtils} = require('./node_modules/kite-common');
+const {SdkTestUtils} = require('./utils/SdkTestUtils');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const {Window} = require('./utils/Window');
+const { v4: uuidv4 } = require('uuid');
+
+/**
+ * Test screen capture sharing check
+ * 1. Turn on content sharing test video and verify that the other participant can see the screen
+ * 2. Turn off screen share and verify that it is off for the other participant
+ */
+class ContentShareVideoTest extends SdkBaseTest {
+  constructor(name, kiteConfig) {
+    super(name, kiteConfig, "ContentShareVideoTest");
+  }
+
+  async runIntegrationTest() {
+    const session = this.seleniumSessions[0];
+    const test_attendee_id_1 = uuidv4()
+    const test_attendee_id_2 = uuidv4();
+    const test_window_1 = await Window.existing(session.driver, "TEST1");
+    const test_window_2 = await Window.openNew(session.driver, "TEST2");
+
+    await test_window_1.runCommands(async () => await SdkTestUtils.addUserToMeeting(this, test_attendee_id_1, session));
+    await test_window_2.runCommands(async () => await SdkTestUtils.addUserToMeeting(this, test_attendee_id_2, session));
+
+    await test_window_2.runCommands(async () => await RosterCheck.executeStep(this, session, 2));
+    await test_window_1.runCommands(async () => await RosterCheck.executeStep(this, session, 2));
+
+    //Turn on Content Share
+    await test_window_1.runCommands(async () => await ClickContentShareVideoTestButton.executeStep(this, session, "ON"));
+    await TestUtils.waitAround(5000);
+    await test_window_1.runCommands(async () => await RosterCheck.executeStep(this, session, 3));
+    await TestUtils.waitAround(5000);
```

Comment:
There should be no need to wait here too. You only need to wait when you turn on off content share since it takes time for the content attendee to join and leave.

Reply:
I added this wait command here because as we know, it takes a while for the content share video to appear on FF browser.
Will remove the other waitAround();